.github/workflows/release.yaml

```diff
@@ -0,0 +1,56 @@
+name: create release on new tags
+
+on:
+  push:
+    # Publish semver tags as releases.
+    tags: [ '*.*.*' ]
+      # [0-9]+.[0-9]+.[0-9]+
+
+jobs:
+  build-and-push:
+
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v3
+
+      - name: Build binary and make tarball
+        id: build_bin
+        run: |
+          make bin
+          tar -zcvf chart-verifier.tgz -C out/ chart-verifier
+          export TARBALL_PATH=$(realpath chart-verifier.tgz)
+          echo "tarball_path=$TARBALL_PATH"
+
+      - name: Print tag to GITHUB_OUTPUT
+        id: get_tag
+        run: |
+          echo "RELEASE_VERSION=${GITHUB_REF#refs/*/}" >> $GITHUB_OUTPUT
+
+      - name: Set up Python 3.x
+        uses: actions/setup-python@v2
+        with:
+          python-version: '3.9'
+
+      - name: Set up Python scripts
+        run: |
+          # set up python requirements and scripts on PR branch
+          python3 -m venv ve1
+          cd scripts && ../ve1/bin/pip3 install -r requirements.txt && cd ..
+          cd scripts && ../ve1/bin/python3 setup.py install && cd ..
+
+      - name: Generate release body
+        id: release_body
+        run: ve1/bin/print-release-body
```

Comment:
This is a little unclear - are we just printing the release body or are we storing it. Should be sending this to `GITHUB_OUTPUT`? If we are doing this in-script, I'd love to see that be a flag so that we can run this on our machine without the expectation of running within the GHA environment. E.g. `--to-github-output`

Reply:
I've changed it to print to stdout and use `tee -a` to get it to GITHUB_OUTPUT too.